Integrations/Cloaken/Cloaken.yml

```diff
@@ -0,0 +1,56 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Cloaken
+  version: -1
+configuration:
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Server URL (e.g., https://cloaken.cypherint.com)
+  name: server_url
+  defaultvalue: "https://cloaken.cypherint.com"
+  required: true
+  type: 0
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: do_not_verify_ssl
```

Comment:
should be `insecure` / `unsecure`.
this is required for the mocks to run properly.